algo/src/main/java/com/hartwig/serve/sources/ckb/curation/CkbCurator.java

```diff
@@ -86,4 +102,85 @@ private Variant curate(@NotNull Variant variant) {
 
         return curatedVariant;
     }
+
+    @NotNull
+    public Location curateFacilityName(@NotNull Location location) {
```

Comment:
This function can be package-private (ie drop the `public` keyword). It can't be private because it is used in a test, so would be good to annotate with `VisibleForTesting`